scripts/test_spm.sh

```diff
@@ -20,32 +20,62 @@
 
 set -euo pipefail
 
-# Set have_secrets to true or false.
-have_secrets=false
-
-. ./scripts/check_secrets.sh
-
-if [[ "$have_secrets" == true ]]; then
-    xcodebuild \
-     -project "${DIR}/${SAMPLE}Example.xcodeproj" \
-     -scheme "${SAMPLE}Example (${OS})" \
-     -sdk "${PLATFORM}simulator" \
-     -destination "platform=${OS} Simulator,name=${DEVICE}" \
-     build \
-     test \
-     ONLY_ACTIVE_ARCH=YES \
-     OTHER_SWIFT_FLAGS=${SWIFT_DEFINES} \
-     | xcpretty
+# Check Xcode version when testing watchOS
+if [[ "$TEST" == true && "$OS" == watchOS ]]; then
+    version="$(xcode-select -p)"
+    if [[ "$version" != "/Applications/Xcode_13.0.app" && \
```

Comment:
Please investigate a generalized test for any Xcode >= 12.5. I'd prefer for this not to start breaking with 13.1 etc.

Reply:
Something like https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/build.sh#L97 could help

Reply:
Done ✅